config/database.yml

```diff
@@ -15,6 +15,7 @@
 # gem 'pg'
 #
 default: &default
+  host: localhost
```

Comment:
https://medium.com/@kevinmircovich/pg-connectionbad-could-not-connect-to-server-no-such-file-or-directory-da18a3764a0a